src/main/webapp/js/user-page-loader.js

```diff
@@ -42,7 +42,23 @@ function showMessageFormIfViewingSelf() {
             loginStatus.username == parameterUsername) {
           const messageForm = document.getElementById('message-form');
           messageForm.classList.remove('hidden');
-          document.getElementById('about-me-form').classList.remove('hidden')
+        }
+      });
+}
+
+/**
+* Shows the AboutMe form if the user is logged in and viewing their own page
+*/
+function showAboutFormIfViewingSelf() {
```

Comment:
I'm assuming you are doing this to separate "message form" behavior from "about form" behavior. The one side effect though is you are now making multiple requests to the server for '/login-status'. One way to avoid this is to split the code in a slightly different way. Separate the fetching of the status from the logic about what to do once you have it.  For example consider replacing the 2 show methods in buildUI() below with something that looks like this.

fetchLoginStatus((loginStatus) => {
  if (!loginStatus.isLoggedIn || loginStatus.username != parameterUsername) {
    return;
  }

  showMessageForm();
  showAboutForm();
});